styles.css

```diff
@@ -169,14 +169,18 @@
   }
 
   .lets-connect {
-    font-family: 'Poppins', sans-serif;
+    font-family: "Poppins", sans-serif;
```

Comment:
- [ ]  I see 😃that there is a problem with the contrast color here at the text ```let's connect```, so kindly try to fix this contrast of the text like ```let's connect``` where The page has a minimum contrast ratio of at least 4.5:1 for normal-size text.😊

| Look Here |
| --- |
|   ![image](https://user-images.githubusercontent.com/83360636/216159399-9ac52a9b-4607-4982-89ee-eac6d462868b.png) |

Reply:
> Correction 

|Let's Connect|Test contrast color|
|:---------:|:-------------------:|
|![imagen](https://user-images.githubusercontent.com/90431831/216195634-69ab9722-5e6f-4db4-86a9-8ada281ca076.png)|![imagen](https://user-images.githubusercontent.com/90431831/216195953-f5706ea0-b2ca-460f-9c52-65cb5fdf79c2.png)|